source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,127 @@
+title: Read the Local Master Key from a File
+ref: read-local-master-key-from-file
+level: 4
+content: |
+  First, read the file containing the local master key that you generated in 
+  the :ref:`Create a Master Key <fle-create-a-master-key>` section:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 11
+
+              String path = "master-key.txt";
+
+              byte[] fileBytes = new byte[96];
+
+              try (FileInputStream fis = new FileInputStream(path)) {
+                fileBytes = fis.readAllBytes();
+              } catch (IOException e) {
+                e.printStackTrace();
+              }
+
+              final byte[] localMasterKey = Arrays.copyOf(fileBytes, 96);
+
+
+---
+title: Specify KMS Provider Settings
+ref: specify-kms-provider-settings
+level: 4
+content: |
+  Next, specify the KMS provider settings. Set the provider name to ``local``
+  when using a local master key:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 4,5
+
+              String kmsProvider = "local";
+
+              Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>() {{
+                put(kmsProvider, new HashMap<String, Object>() {{
+                  put("key", localMasterKey);
+                }});
+              }};
+
+
+---
+title: Create a Data Encryption Key
+ref: create-a-data-encryption-key
+level: 4
+content: |
+  Construct a client with the MongoDB connection string and key vault 
+  namespace configuration, and create a data encryption key. The key vault in 
+  this example uses the ``encryption`` database and ``__keyVault`` collection.
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 8,9,12
```

Comment:
Do you think I might be overusing the `emphasize-lines` directive?

Reply:
Nope! Only 3 lines is fine and the highlighted lines here seem relevant.